src/models/gradient_boosting/xgboost_model.py

```
@@ -1165,25 +1161,25 @@ def main():
+        logger.info(f"Traditional MSE: {final_mse:.4f}")
+        logger.info(f"Traditional MAE: {final_mae:.4f}")
+        logger.info(f"Traditional R²: {final_r2:.4f}")
 
         # 5. Feature importance analysis
         logger.info("5. Analyzing feature importance...")
 
         feature_importance = final_model.get_feature_importance("gain")
 
-        logger.info("📊 Top 30 Most Important Features:")
+        logger.info("Top 30 Most Important Features:")
         for idx, row in feature_importance.head(30).iterrows():
```

Ruff (0.14.4)

Loop control variable `idx` not used within loop body

Rename unused `idx` to `_idx`

(B007)